naming/src/main/java/com/alibaba/nacos/naming/misc/GlobalExecutor.java

```diff
@@ -42,141 +50,195 @@
     
     private static final long SERVER_STATUS_UPDATE_PERIOD = TimeUnit.SECONDS.toMillis(5);
     
-    private static ScheduledExecutorService executorService = new ScheduledThreadPoolExecutor(
-            Runtime.getRuntime().availableProcessors() * 2, registerMaster -> {
-        Thread t = new Thread(registerMaster);
-        
-        t.setDaemon(true);
-        t.setName("com.alibaba.nacos.naming.timer");
-        
-        return t;
-    });
-    
-    private static ScheduledExecutorService taskDispatchExecutor = new ScheduledThreadPoolExecutor(
-            Runtime.getRuntime().availableProcessors(), taskDispatch -> {
-        Thread t = new Thread(taskDispatch);
-        
-        t.setDaemon(true);
-        t.setName("com.alibaba.nacos.naming.distro.task.dispatcher");
-        
-        return t;
-    });
-    
-    private static ScheduledExecutorService dataSyncExecutor = new ScheduledThreadPoolExecutor(
-            Runtime.getRuntime().availableProcessors(), dataSync -> {
-        Thread t = new Thread(dataSync);
-        
-        t.setDaemon(true);
-        t.setName("com.alibaba.nacos.naming.distro.data.syncer");
-        
-        return t;
-    });
-    
-    private static ScheduledExecutorService notifyServerListExecutor = new ScheduledThreadPoolExecutor(1, notifyServer -> {
-        Thread t = new Thread(notifyServer);
-        
-        t.setDaemon(true);
-        t.setName("com.alibaba.nacos.naming.server.list.notifier");
-        
-        return t;
-    });
-    
-    private static final ScheduledExecutorService SERVER_STATUS_EXECUTOR = new ScheduledThreadPoolExecutor(1, serverStatus -> {
-        Thread t = new Thread(serverStatus);
-        t.setName("nacos.naming.status.worker");
-        t.setDaemon(true);
-        return t;
-    });
+    public static final int DEFAULT_THREAD_COUNT =
+            Runtime.getRuntime().availableProcessors() <= 1 ? 1 : Runtime.getRuntime().availableProcessors() / 2;
+    
+    private static final ScheduledExecutorService NAMING_TIMER_EXECUTOR = ExecutorFactory.Managed
+            .newScheduledExecutorService(ClassUtils.getCanonicalName(NamingApp.class),
+                    Runtime.getRuntime().availableProcessors() * 2,
+                    new NameThreadFactory("com.alibaba.nacos.naming.timer"));
```

Comment:
Use public final static String variable to represent the const value "com.alibaba.nacos.naming.timer" is better? The same is as below.

Reply:
If set to public, there is no need to provide a method